setup.py

```diff
@@ -105,20 +115,16 @@
             f"snowflake-connector-python[secure-local-storage]{CONNECTOR_DEPENDENCY_VERSION}",
         ],
         "development": [
-            "pytest<8.0.0",  # check SNOW-1022240 for more details on the pin here
-            "pytest-cov",
-            "coverage",
-            "sphinx==5.0.2",
-            "cachetools",  # used in UDF doctest
-            "pytest-timeout",
-            "pre-commit",
+            *DEVELOPMENT_REQUIREMENTS,
         ],
         "modin-development": [
+            *PANDAS_REQUIREMENTS,
+            *DEVELOPMENT_REQUIREMENTS,
+            f"modin{MODIN_DEPENDENCY_VERSION}",
```

Comment:
```suggestion
            *DEVELOPMENT_REQUIREMENTS,
            *MODIN_REQUREMENTS,
```

should we do this and save `MODIN_REQUIREMENTS` and use it in the `"modin"` requirements?

Reply:
Yep good idea done.